browser/search_engines/search_engine_provider_service_factory.cc

```diff
@@ -103,4 +103,5 @@ void SearchEngineProviderServiceFactory::RegisterProfilePrefs(
   registry->RegisterBooleanPref(kUseAlternativeSearchEngineProvider, false);
   registry->RegisterBooleanPref(prefs::kDefaultSearchProviderByExtension,
                                 false);
+  registry->RegisterBooleanPref(prefs::kAddOpenSearchEngines, false);
```

Comment:
Wanted to share where we got confirmation from @diracdeltas that this is OK to set to false by default:
https://github.com/brave/brave-browser/issues/14496#issuecomment-797733599